suro-elasticsearch/src/main/java/com/netflix/suro/sink/elasticsearch/ElasticSearchSink.java

```diff
@@ -259,23 +275,29 @@ public void run() {
                 HttpResponse response = null;
                 try {
                     response = client.executeWithLoadBalancer(request.first());
+                    stopwatch.stop();
```

Comment:
I would like to have the timer only measures the time for successful execution which will help us to set appropriate timeout in the future. If NullPointerException is thrown, the timer should not be updated otherwise the metric does not correctly reflect the time it takes to get response from server.